frontend/app/controllers/Event.scala

```diff
@@ -176,7 +172,7 @@ class Event(
     Found(addEventBriteGACrossDomainParam(eventUrl)).withCookies(Cookie(eventCookie(event), "", Some(3600)))
   }
 
-
+  @deprecated("Guardian Live Events are not configured to use this anymore, and host their own thankyou page on EventBrite.")
```

Comment:
I noticed a message I think it was in chat or email saying we are not limiting the number of tickets people buy, which is affecting paid ticket sales.  I noticed this is because this ThankYou page was writeing in zuora that a free ticket had been used.  Now when we query zuora everyone always has a full complement of free tickets as this isn't called any more.